# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../apcupsd/apcupsd-3.14.10-manify-workaround.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Workaround to not bail-out when trying to format a manpage into plain text.

--- apcupsd-3.14.10/autoconf/targets.mak.orig	2012-06-25 18:10:25.737904816 +0200
+++ apcupsd-3.14.10/autoconf/targets.mak	2012-06-25 18:11:05.222633602 +0200
@@ -293,5 +293,5 @@
 # Format a manpage into plain text
 define MANIFY
 	@$(ECHO) "  MAN  " $(1) -\> $(2)
-	$(V)man ./$(1) | col -b > $(2)
+	$(V)less -X ./$(1) 2>/dev/null | col -b > $(2)
 endef
